<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRandomUI</class>
 <widget class="QWidget" name="EditRandomUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IN2PS</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="m_gbCentral">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>12</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Configuración Número Aleatorio</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Límite inferior</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="In2LineEdit" name="m_limiteInferior">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Límite superior</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="In2LineEdit" name="m_limiteSuperior">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Nº.Cifras</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="In2LineEdit" name="m_numCifras">
          <property name="text">
           <string>10</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Secuencia
generada</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTextEdit" name="m_Generated">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_Aplicar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Aplicar</string>
       </property>
       <property name="icon">
        <iconset resource="../../Xaar/InkjetModule/InkjetModule/shared.qrc">
         <normaloff>:/res/images/reload.png</normaloff>:/res/images/reload.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Aceptar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../../Xaar/InkjetModule/InkjetModule/shared.qrc">
         <normaloff>:/res/images/Ok_32x32.png</normaloff>:/res/images/Ok_32x32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Cancelar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../../Xaar/InkjetModule/InkjetModule/shared.qrc">
         <normaloff>:/res/images/Cancel_32x32.png</normaloff>:/res/images/Cancel_32x32.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>In2LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">in2lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Xaar/InkjetModule/InkjetModule/shared.qrc"/>
 </resources>
 <connections/>
</ui>
